ethsigner/signer/hashicorp/src/main/java/tech/pegasys/ethsigner/signer/hashicorp/HashicorpSubCommand.java

```diff
@@ -132,6 +161,20 @@ public boolean isTlsEnabled() {
     return Optional.ofNullable(tlsKnownServerFile);
   }
 
+  private String readTokenFromFile(final Path path) {
+    try (final Stream<String> stream = Files.lines(path)) {
+      return stream
+          .findFirst()
+          .orElseThrow(
+              () ->
+                  new TransactionSignerInitializationException(
+                      String.format(AUTH_FILE_ERROR_MSG_FMT, path.toString())));
+    } catch (final IOException e) {
```

Comment:
Does FileNotFoundException also need to be handled?

Reply:
FileNotFoundException is a subtype fo IOException, so this will catch it - but maybe not give the best error msg?

Reply:
Probably won't be the best error msg, but will probably be sufficient